docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
+- HTTP Archive (sql, csv)
+- Google Storage (har)
+- BigQuery (har)
+
```

Comment:
I think there should be a distinction between README style docs where we give tips and instructions for how to use HA, and /docs where we describe how the system works. Here, I think it's valuable to be complete and cover the different ways HA persists the data because if not here then where else?